# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
# Commit: bca550582491
# Full Hash: bca550582491193228f12d9b28c276c52fce80db
# Author: Jeff Muizelaar <jmuizelaar@mozilla.com>
# Date: 2021-06-26 09:46:54
# Regressor Bug: 1718334
# File Overlap Count: 1
# Description:
#   Bug 1718334 - RELEASE_ASSERT the last map succeeded. r=jgilbert
#   
#   The RELEASE_ASSERT below is failing so this will help
#   us narrow down which Map() call it might be.
#   
# ==============================================================================

diff -r bfae6deb1397 -r bca550582491 gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
--- a/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Fri Jun 25 20:53:59 2021 +0000
+++ b/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Fri Jun 25 20:56:14 2021 +0000
@@ -297,6 +297,7 @@
       }
       hr = context->Map(mRenderCompositor->mCurrentStagingTexture, 0,
                         D3D11_MAP_READ_WRITE, 0, &mappedSubresource);
+      MOZ_RELEASE_ASSERT(SUCCEEDED(hr));
     }
   }
   MOZ_RELEASE_ASSERT(SUCCEEDED(hr));
